share_app/src/main/java/org/odk/share/adapters/FormsAdapter.java

```diff
@@ -29,16 +29,20 @@
 
 public class FormsAdapter extends CursorRecyclerViewAdapter<FormsAdapter.FormHolder> {
 
+    private final InstancesDao instancesDao;
+    private final TransferDao transferDao;
     @Nullable
-    private LinkedHashSet<Long> selectedForms;
+    private final LinkedHashSet<Long> selectedForms;
```

Comment:
Any reason for making it final?

Reply:
To ensure that the variable is never mutated again in the class.